### PR TITLE
app-doc/kicad-doc: Bump to stable release 4.0.0 version

### PR DESCRIPTION
**Pull request by zpuskas**
@idella 
@jlec 
@gentoo/proxy-maint

### CODE REVIEW

File: app-doc/kicad-doc/kicad-doc-4.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils vcs-snapshot
+
+DESCRIPTION="Electronic Schematic and PCB design tools manuals."
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.0.ebuild]: Drop ```.```at the end. This is not a sentence.

File: app-doc/kicad-doc/kicad-doc-4.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	app-text/dblatex
+	>=app-text/po4a-0.45
+	>=sys-devel/gettext-0.18
+	dev-util/source-highlight
+	dev-perl/Unicode-LineBreak
+	linguas_ja? ( media-fonts/vlgothic )"
+RDEPEND=""
+
+src_prepare() {
+	sed "s|KICAD_DOC_INSTALL_PATH share/doc/kicad|KICAD_DOC_INSTALL_PATH share/doc/kicad-${PV}|g" -i CMakeLists.txt || die "sed failed"
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.0.ebuild]: Could you break the line here?

Comment (idella) [inline app-doc/kicad-doc/kicad-doc-4.0.0.ebuild]: agreed, way too long

**Comment by idella** [comment]
sed "s|KICAD_DOC_INSTALL_PATH share/doc/kicad|KICAD_DOC_INSTALL_PATH share/doc/kicad-${PV}|g" -i CMakeLists.txt \

This is so long it needs a bash var to help it, like INSPATH="KICAD_DOC_INSTALL_PATH"
then use ${INSPATH} which can be bash expanded
and break 

/kicad-${PV}|g" -i CMakeLists.txt \

into 
```ebuild
/kicad-${PV}|g" \
(/t)-i CMakeLists.txt \
```

**Comment by idella** [comment]
merged

**Pull request closed without merging**